tensorflow/contrib/receptive_field/python/util/receptive_field.py

```diff
@@ -322,22 +321,34 @@ def __init__(self, size, stride, padding):
     self.stride = np.asarray(stride)
     self.padding = np.asarray(padding)
 
-  def compute_input_coordinates(self, x, axis=None):
+  def compute_input_center_coordinates(self, x, axis=None):
```

Comment:
I am a little bothered by the fact that output feature coordinates are "x" and input center coordinates are "y". Ie, "y" at the input and "x" at the output, which is a little weird. I think swapping the two notations would be better for clarity, and it shouldn't be too hard since there not that many cases. Can you please do it?